marklogic-data-hub-central/ui/src/components/queries/managing/manage-query-modal/manage-query.tsx

```diff
@@ -32,6 +35,10 @@ const QueryModal = (props) => {
         getQueries();
```

Comment:
this is inefficient because you are calling getQueries() anytime these 3 booleans change. If I click "cancel" on delete or edit, or main modals, it will do a network request.

Reply:
Fixed. Thanks.